src/Routes/JoinRoom/index.js

```diff
@@ -0,0 +1,27 @@
+import RoomService from '../../Services/Room';
+
+const toJSON = (record) => {
+  delete record._Model; // remove circular structure and system info
+  return record;
+};
+
+const joinRoom = async (req, res) => {
+  const { username } = req.body;
+  const { id } = req.params;
+
+  if (!username) {
+    return res.status(400).send({
+      message: 'Username must be in to the body and dont be empty!',
+    });
+  }
+
+  const room = await RoomService.joinRoom({ id, username });
+  return res.status(201).send({ room: toJSON(room) });
```

Comment:
`201` HTTP kodu yeni bir kayıt oluşturulduğunda kullanıcıya dönülür. Normal bir işlem başarılı olduğunda `200` dönmemiz yeterli olacaktır.

**Daha detaylı bigi için**:
[https://httpstatuses.com/](https://httpstatuses.com/)
[https://developer.mozilla.org/en-US/docs/Web/HTTP/Status](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)